### DISCUSSION TITLE
How to force Ikev2 client to use their local internet for browsing instead of remote server.

### DISCUSSION THREAD

**Discussion by RixwanSaleem** [strongswan/strongswan#295] [General]
How to force Ikev2 client to use their local internet for browsing instead of remote server 

default via 172.31.32.1 dev eth0 proto dhcp metric 100 
172.31.32.0/20 dev eth0 proto kernel scope link src 172.31.42.77 metric 100

**Comment by Thermi** [comment]
Depends on the negotiated TS and what the client ultimately decides to do.
Routes are only tangentially related to the issue, as in, policy based tunnels will work if the host uses the right source IP for new connections, and route based tunnels will only work if, well, the route goes over the tunnel (and evidently the traffic is allowed by the negotiated TS).
